pages/having-a-baby.js

```diff
@@ -107,18 +107,18 @@ export default function lifejourney({ journeysData }) {
 
           {/* Journey Stages Section */}
 
-          <div className="container flex flex-col w-full bg-gray-200 md:flex-row align-items-center mt-4">
-            <div className="p-2">
-              <h2 className="text-h3-tall">
+          <div className="container flex flex-row w-full bg-gray-200 xxs:flex-col align-items-center mt-4">
```

Comment:
```suggestion
          <div className="container flex flex-row w-full bg-gray-light-200 xxs:flex-col align-items-center mt-4">
```
Change the background color here so the stages picture blends in